### ISSUE TITLE
Bloc ??

### ISSUE THREAD

**Issue by iampato** [droidconKE/droidconke2020_flutter#1]
The state management solution ???

**Comment by danvick** [comment]
Yes. What are your thoughts?

**Comment by iampato** [comment]
i see you mix with provider  !

**Comment by danvick** [comment]
Yeah, as it's written in the README, we're also using Provider, but only as a "dependency injector" and nothing else.
